dcos-log/api/v2/handlers.go

```diff
@@ -299,16 +297,16 @@ func redirectURL(id *nodeutil.CanonicalTaskID, file, RawQuery string, browse, do
 		id.FrameworkID, executorID, taskID)
 
 	if isPod {
-		taskLogURL += fmt.Sprintf("/tasks/%s", id.ID)
+		taskLogURL += path.Join("/tasks/", id.ID)
```

Comment:
small nit, i don't think you need the trailing `/` on `"/tasks/"` ... same elsewhere when joining paths.